.circleci/config.yml

```diff
@@ -961,7 +948,7 @@ workflows:
           xcode_version: '14.3.0'
           <<: *release-branches
       - installation-tests-receipt-parser:
-          xcode_version: '14.1.0'
+          xcode_version: '14.2.0'
```

Comment:
Looks like we are still running some of these in `14.2.0`. Should  we be using `14.3.0`?

Reply:
Oh this can be 14.3

But the others do need 14.2 so they run on macOS 12, which is required to install those runtimes. I added a comment for that.